fedimint-core/src/admin_client.rs

```diff
@@ -216,6 +216,15 @@ pub enum ServerStatus {
     ConsensusRunning,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct FullStatus {
+    pub our_epoch: u64,
+    pub last_epoch_by_peer: HashMap<PeerId, u64>,
+    pub peers_offline: u64,
+    pub peers_not_synchronized: u64,
+    pub status: ServerStatus,
+}
```

Comment:
FullStatus vs `statsu` vs `ServerStatus`. Too much status maybe.

Reply:
Naming stuff is hard :sweat_smile:

Reply:
I would combine this with the existing status endpoint if possible, it would only be available when running consensus though.